src/app/api/volunteer/auth/password-reset/validate/route.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  deletePasswordResetField,
+  getPasswordByToken,
+} from "@server/mongodb/actions/PasswordReset";
+import APIWrapper from "@server/utils/APIWrapper";
+
```

Comment:
Add `export const dynamic = "force-dynamic";` here. The absence of this line throws a "Dynamic server usage" error in production for GET requests that take in search parameters.